LayoutTests/fast/pdf-plugin-destruction-dispatches-print-event-crash.html

```
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<p>PASS if no crash.</p>
+<embed id="embed" style="top: 200vmin; visibility: hidden;" type="application/pdf" src="about:blank"></embed>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </embed> ]

(tag-pair)